docs/environment-variables.md

```diff
@@ -53,6 +53,7 @@
 | AUTH_JWT_CUSTOM_CLAIMS                                |                                                                                                                                                                                                                                         |                              |
 | AUTH_WEBAUTHN_ENABLED                                 | When enabled, passwordless Webauthn authentication can be done via device supported strong authenticators like fingerprint, Face ID, etc.                                                                                               | false                        |
 | AUTH_WEBAUTHN_RP_NAME                                 | Relying party name. Friendly name visual to the user informing who requires the authentication. Probably your app's name.                                                                                                               |                              |
+| AUTH_WEBAUTHN_RP_ID                                   | Relying party id.                                                                                                                                                                                                                       |                              |
```

Comment:
maybe add a note that it defaults to client url

Reply:
yep will add now